packages/nextjs/components/scaffold-eth/Address.tsx

```diff
@@ -87,12 +86,8 @@ export const Address = ({ address, disableAddressLink, format, size = "base" }:
           size={(blockieSizeMap[size] * 24) / blockieSizeMap["base"]}
         />
       </div>
-      {disableAddressLink ? (
+      {disableAddressLink || targetNetwork.id === hardhat.id ? (
         <span className={`ml-1.5 text-${size} font-normal`}>{displayAddress}</span>
-      ) : targetNetwork.id === hardhat.id ? (
-        <span className={`ml-1.5 text-${size} font-normal`}>
-          <Link href={blockExplorerAddressLink}>{displayAddress}</Link>
-        </span>
```

Comment:
1. To play around on localnetwork (hardhat | foundry) user needs a local chain running.
2. If the user local chain is running, the local blockexplorer should also work nicely 

So do we really want to remove this, functionality ?

Reply:
I think it is a bit weird to have it in abi.ninja. Maybe we deploy that as a separate webapp for people to use with any local blockchain node?
like chain.vision or something : D

Reply:
Yup yup agree!